docs/guides/1238-making-reusable-package.md

```diff
@@ -0,0 +1,188 @@
+---
+slug: /1238/making-reusable-package
+displayed_sidebar: "0.2"
+---
+
+# Making reusable packages
+
+Whilst splitting your plan into several files is a good idea, you will sometimes need to create standalone packages aiming to be reusable and shared. Let's explore how to do that.
+
+## Dagger project vs packages
```

Comment:
There's actually 3 concepts at play here[^1]:

- **CUE Package:** directory with cue files, each including a `package` definition on top, making it importable (e.g, `universe.dagger.io/docker` is a package, `universe.dagger.io/docker/cli` is another package);
- **CUE Module:** directory with a `cue.mod` directory which makes for the prefix/root of importable packages (e.g, `universe.dagger.io` is a module);
- **Dagger Project**: a CUE module that includes dagger plans and dependencies for running the `dagger` CLI.

We should link to CUE's documentation for the first two: https://cuelang.org/docs/concepts/packages/

They're their own thing, so I don't thing the "vs" is adequate. Use a comma instead: _"Packages, modules, and Dagger projects"_

[^1]: https://github.com/dagger/dagger/issues/2186#issuecomment-1119529251